index.html

```diff
@@ -209,40 +216,65 @@ <h4 class="lang-title">Skils</h4>
       </div>
     </section>
 
-    <section class="form-section">
+    <section class="form-section container">
       <div class="form-container">
-        <h2 class="form-title">
-          I'm always interested in hearing about new projects, so if you'd like
-          to chat please get in touch.
-        </h2>
-        <form action="https://formspree.io/f/xqkjapkj" method="post">
-          <input
-            type="text"
-            maxlength="30"
-            name="name"
-            id="user-name"
-            required
-            placeholder="Full name"
-          />
-          <input
-            type="email"
-            name="user-email"
-            id="user-email"
-            required
-            placeholder="Email address"
-          />
-          <textarea
-            name="user-text"
-            id="user-message"
-            maxlength="500"
-            cols="30"
-            rows="10"
-            placeholder="Enter text here"
-            required
-          >
-          </textarea>
-          <button type="submit" class="btn">Get in touch</button>
-        </form>
+        <div class="text-form">
+          <div class="fort-heading">
+            <h2 class="form-title">
+              I'm always interested in hearing about new projects, so if you'd
+              like to chat please get in touch.
+            </h2>
+          </div>
+          <div class="form">
+            <form
+              action="https://formspree.io/f/xqkjapkj"
+              method="post"
+              class="form"
+            >
+              <input
+                type="text"
+                maxlength="30"
+                name="name"
+                id="user-name"
+                required
+                placeholder="Full name"
+              />
+              <input
+                type="email"
+                name="user-email"
+                id="user-email"
+                required
+                placeholder="Email address"
+              />
+              <textarea
```

Comment:
- [ ] [REQUIRED] This is required because you introduce some irregularities as you implemented the request of the previous reviewer. The bold text on the left column is not properly aligned to the left and as a result, the input section has some of its parts hidden. Please screenshot below. Kindly fix these issues.

Figma Design | Your Design
----- | ------
![image](https://user-images.githubusercontent.com/84156714/186792005-2cbf4431-183b-421b-ade4-6d47af362309.png) | ![image](https://user-images.githubusercontent.com/84156714/186792079-aec16861-7ccc-4af0-88e9-fce91ce5dc4e.png)